app/src/main/kotlin/compiler/syntax/LanguageGrammar.kt

```diff
@@ -32,13 +32,21 @@ object LanguageGrammar {
         val callArgs1 = getProduction("CALL_ARGS", "({nE_EXPR}({tCOMMA}$NLS{nE_EXPR})*({tCOMMA}$NLS{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR})*)?")
         val callArgs2 = getProduction("CALL_ARGS", "{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR}({tCOMMA}$NLS{nE_EXPR}{tASSIGNMENT}$NLS{nE_EXPR})*")
 
+        // TODO: add "wewnątrz" handling.
+        // val generatorIteration = getProduction("GENERATOR_ITERATION", "{tFOR_EACH}$NLS{tIDENTIFIER}$NLS{tCOLON}$NLS{nTYPE}$NLS{tFROM}$NLS{tIDENTIFIER}{tLEFT_PAREN}$NLS{nCALL_ARGS}{tRIGHT_PAREN}")
```

Comment:
I need to consult this with someone who understands well the role of newlines in LL(1). Already asked on discord for such a person.

Reply:
@bebidek @kamgal00